src/sass/components/cards.scss

```diff
@@ -72,32 +78,49 @@
             @include title-small;
             padding: $padding-card;
             text-transform: capitalize;
+            background: $color-card-background;
         }
 
         p {
             @include text-normal;
-            padding: 0 $padding-card ($size-card-arrow + 2 * $padding-card);
+            line-height: $line-height-small;
+            padding: 0 $padding-card $size-card-arrow + 2 * $padding-card;
+        }
+
+        &_with-footer {
+            p {
+                padding-bottom: $padding-card;
+            }
         }
 
         footer {
-            position: absolute;
+            @include valign-flex-center;
             bottom: $padding-card;
-            left: $padding-card;
-            right: $size-card-arrow + $padding-card;
+            height: $size-card-author-image;
+            width: 100%;
+            margin: 0 $size-card-arrow + 2 * $padding-card $padding-card $padding-card;
+            align-self: flex-end;
+            min-height: $size-card-arrow;
```

Comment:
Move `min-height` closer to `height`. I'll try to get some sort of time allocated for documenting and approving a stack of tools / linter / whatever to help us doing this automatically.